### PR TITLE
fix: deprecate KEYCLOAK_ADMIN and KEYCLOAK_ADMIN_PASSWORD

### LINKED ISSUE #30658: Remove or deprecate old KEYCLOAK admin properties
KEYCLOAK_ADMIN, KEYCLOAK_ADMIN_PASSWORD, and the associated system properties are no longer needed.

### PR DESCRIPTION
**Pull request by shawkins**
closes: #30658

### CODE REVIEW

File: docs/documentation/upgrading/topics/changes/changes-26_0_0.adoc [hunk 1]
```diff
@@ -71,3 +71,7 @@ If you are migrating from previous versions where any of the following settings
 * `connectionPoolingDebug`
 
 For more details, see link:{adminguide_link}#_ldap_connection_pool[Configuring the connection pool].
+
+= Admin Bootstrapping
+
+The environment variables KEYCLOAK_ADMIN and KEYCLOAK_ADMIN_PASSWORD have been deprecated. You should use KC_BOOTSTRAP_ADMIN_USERNAME and KC_BOOTSTRAP_ADMIN_PASSWORD instead. These are also general options, so they may be specified via the cli or other config sources, for example --bootstrap-admin-username=admin.
```
Comment (jonkoops) [inline docs/documentation/upgrading/topics/changes/changes-26_0_0.adoc:77]: Maybe add some backticks for formatting text?

```suggestion
The environment variables `KEYCLOAK_ADMIN` and `KEYCLOAK_ADMIN_PASSWORD` have been deprecated. You should use `KC_BOOTSTRAP_ADMIN_USERNAME` and `KC_BOOTSTRAP_ADMIN_PASSWORD` instead. These are also general options, so they may be specified via the cli or other config sources, for example `--bootstrap-admin-username=admin`.
```

**Review by jonkoops — DISMISSED** [review]
LGTM

**Review by jonkoops — APPROVED** [review]

**Review by Pepo48 — APPROVED** [review]
LGTM. Thanks, @shawkins

**Review by ahus1 — APPROVED** [review]
Approving based on @Pepo48 and @jonkoops' reviews.

**Pull request merged**